doofinder.php

```diff
@@ -2037,4 +2040,27 @@ private function getInstallUrl($region)
     {
         return str_replace('{region}', $region, 'https://{region}-plugins.doofinder.com/install');
     }
+
+    private function checkAndSetScriptFlag()
+    {
+        if (Configuration::get('DF_UNIQUE_SCRIPT') === false && class_exists('DoofinderApiUniqueScript')) {
+            $installtaion_id = Configuration::get('DF_INSTALLATION_ID');
```

Comment:
```suggestion
            $installationId = Configuration::get('DF_INSTALLATION_ID');
```

Also if it were possible, try to use camelCase instead, unlike WordPress, Prestashop seems to use camelCase :smile: